### ISSUE TITLE
Integration tests failing in 0.9.1

### ISSUE THREAD

**Issue by drcrallen** [druid-io/druid#3115]
Integration tests are failing when I run them with:

````
Tests run: 6, Failures: 3, Errors: 0, Skipped: 0, Time elapsed: 1,742.423 sec <<< FAILURE! - in TestSuite
testUnionQuery(io.druid.tests.indexer.ITUnionQueryTest)  Time elapsed: 402.581 sec  <<< FAILURE!
com.metamx.common.ISE: one or more queries failed
	at io.druid.tests.indexer.ITUnionQueryTest.testUnionQuery(ITUnionQueryTest.java:119)

testTwitterQueriesFromFile(io.druid.tests.query.ITTwitterQueryTest)  Time elapsed: 119.515 sec  <<< FAILURE!
com.metamx.common.ISE: one or more queries failed
	at io.druid.tests.query.ITTwitterQueryTest.testTwitterQueriesFromFile(ITTwitterQueryTest.java:62)

testWikipediaQueriesFromFile(io.druid.tests.query.ITWikipediaQueryTest)  Time elapsed: 38.084 sec  <<< FAILURE!
com.metamx.common.ISE: one or more queries failed
	at io.druid.tests.query.ITWikipediaQueryTest.testWikipediaQueriesFromFile(ITWikipediaQueryTest.java:62)


Results :

Failed tests:
  ITUnionQueryTest.testUnionQuery:119 » ISE one or more queries failed
  ITTwitterQueryTest.testTwitterQueriesFromFile:62 » ISE one or more queries fai...
  ITWikipediaQueryTest.testWikipediaQueriesFromFile:62 » ISE one or more queries...

Tests run: 6, Failures: 3, Errors: 0, Skipped: 0
```

I have a bad history of not being able to have integration tests run, so if someone could confirm/refute if the integration tests pass I would appreciate it

**Comment by gianm** [comment]
On which branch?

**Comment by drcrallen** [comment]
0.9.1 as per milestone. Will update description

**Comment by drcrallen** [comment]
@nishantmonu51 Are you able to check this?

**Comment by nishantmonu51** [comment]
checking

**Comment by nishantmonu51** [comment]
It was failing due to #3119

**Comment by drcrallen** [comment]
hey, an actual bug. Nice. Good job integration tests. and thanks @nishantmonu51

**Comment by nishantmonu51** [comment]
fixed by #3119.
